zgw/besluiten-api/src/main/kotlin/com/ritense/besluitenapi/BesluitenApiPlugin.kt

```diff
@@ -99,28 +104,32 @@ class BesluitenApiPlugin(
         @PluginActionProperty uiterlijkeReactieDatum: LocalDate?,
         @PluginActionProperty createdBesluitUrl: String?,
     ) {
-
         val documentId = UUID.fromString(execution.businessKey)
         val zaakUrl = zaakUrlProvider.getZaakUrl(documentId)
 
-        logger.debug { "Creating besluit for zaak $zaakUrl of type $besluittypeUrl" }
+        withLoggingContext(
+            "documentId" to documentId.toString(),
+            "zaakUrl" to zaakUrl.toString(),
+        ) {
+            logger.info { "Creating besluit for zaak $zaakUrl of type $besluittypeUrl" }
```

Comment:
We don't get this line when using the create method below. I think we should move it.